spec/models/event_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe Event, type: :model do
+  let!(:user) { create(:user) }
+
+    describe "attributes" do
+      it { expect(subject.attributes).to include('title', 'description') }
+    end
+
+    describe "validation" do
+      it { is_expected.to validate_presence_of(:title) }
+      it { is_expected.to validate_presence_of(:description) }
+      it { is_expected.to validate_presence_of(:event_time) }
+      it { is_expected.to validate_presence_of(:event_date) }
+      it { is_expected.to validate_length_of(:description).is_at_most(500) }
+      it { is_expected.to validate_length_of(:title).is_at_most(35) }
+    end
+
+    describe "relations" do
```

Comment:
Test `has_and_belongs_to_many` relation. Read about it [here](https://www.rubydoc.info/github/thoughtbot/shoulda-matchers/master/Shoulda%2FMatchers%2FActiveRecord:have_and_belong_to_many)